docs/package.json

```diff
@@ -42,8 +43,7 @@
     "postcss": "8.4.14",
     "tailwindcss": "3.1.6",
     "ts-json-schema-generator": "0.98.0",
-    "typescript": "4.7.4",
-    "webpack": "5.65.0"
```

Comment:
I got excited and then realized this was going to make zero difference as transitives are going to add it back.